### PR TITLE
calculator functionality

### PR DESCRIPTION
**Pull request by sjdev2212**
In this feature I : 
Copied files given by microverse and place them inside a logic folder.
Added functionality to the calculator using those files.
Checked linters.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,35 +1,60 @@
 import React from 'react';
+import calculate from './logic/calculate';
 
 class Calculator extends React.PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      obj: {
+        total: '541',
```
Comment (omar-labana) [inline src/components/Calculator.js]: - [x] Please clear this static value from the internal state. The output of the calculator when first launched is `541`.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,35 +1,60 @@
...
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '5')}>5</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '6')}>6</button>
+            <button type="button" className="btn operation" onClick={() => this.pressedButton(obj, '-')}>-</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '1')}>1</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '2')}>2</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '3')}>3</button>
+            <button type="button" className="btn operation" onClick={() => this.pressedButton(obj, '+')}>+</button>
+            <button type="button" className="btn number zero" onClick={() => this.pressedButton(obj, '0')}>0</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '.')}>.</button>
+            <button type="button" className="btn equal operation" onClick={() => this.pressedButton(obj, '=')}>=</button>
```
Comment (omar-labana) [inline src/components/Calculator.js]: - [ ] I would suggest practicing passing Props to child components of the Calculate.js component, you should create a button component then pass down the button operation name along with the clickhandler to it.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @sj1978, 

Excellent work so far! 👌
The repo is quite professional, and the UI is looking amazing. ✔️👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @sj1978 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**